storage/api-docs/api-docs.json

```diff
@@ -18,7 +18,7 @@
             "description": "Production Server"
         },
         {
-            "url": "http://localhost:8000/api/v1",
+            "url": "http://localhost:8000/api/v1/api/v1",
```

Comment:
Your .env seems to be the old version. Please see 9025efc346af73ae6a94ae2c99f7b994930440cd